software-project-management-plan.md

```diff
@@ -6,17 +6,129 @@
 ## VERSION HISTORY
 | Version # | Implemented By | Revision Date | Approved By | Approval Date | Reason |
 |-----------|----------------|---------------|-------------|---------------|--------|
+| 1.0       | Mia            | 13/11/23      | Mia         | 13/11/23      | Added Template. |
+| 1.1       | Cody           | 09/24/23      | Cody        | 09/25/23      | Added introduction and project organization. |
```

Comment:
For the approval date are we doing day month year (?) and the cody entry seems weird lol

Reply:
technically being approved by me will get changed in the end to be by everyone. but I updated the date. also I hate month day year but I changed it since thats what we have gone with so far lol